build_tools/docker/dockerfiles/nvidia.Dockerfile

```diff
@@ -11,12 +11,12 @@
 # We use .deb files that we host because we have to pin the version and packages
 # routinely dissapear from the Ubuntu apt repositories. The versions need to be
 # compatible with the host driver (usually <= host driver version).
-ARG NVIDIA_GL_DEB="libnvidia-gl-530_530.41.03-0ubuntu0.20.04.2_amd64.deb"
-ARG NVIDIA_COMPUTE_DEB="libnvidia-compute-530_530.41.03-0ubuntu0.20.04.2_amd64.deb"
-ARG NVIDIA_COMMON_DEB="libnvidia-common-530_530.41.03-0ubuntu0.20.04.2_all.deb"
```

Comment:
The NVIDIA driver on host is still 530.41

```
NVIDIA-SMI 530.41.03              Driver Version: 530.41.03    CUDA Version: 12.1
```

Do we need to upgrade them?

Reply:
I think it's fine. CUDA 12.2.x requires >=525.60.13.

Reply:
Nah. It turns out we also need to update the base VM: https://github.com/openxla/iree/actions/runs/5981136086/job/16228796396?pr=14833#step:7:164 :(

@pzread could you help with that? Thanks!

Reply:
I'll do it early this week

Reply:
I set up testing runners with the new NVIDIA driver. The tests can run now (but some of them are failing): https://github.com/openxla/iree/actions/runs/5981136086/job/16283843823?pr=14833

FYI, I put these lines in the PR description so CI will run with testing runners and only run `test_gpu` job to save time. We will need to remove it before submit for a complete CI run.

```
runner-env: testing
ci-exactly: build_all, test_gpu
```

Reply:
The nvidia driver on all runners has been upgraded to `535.86.05`

Reply:
Nice, thanks @pzread! I'll try it again now.